CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift

```diff
@@ -395,7 +400,8 @@ public let TYPE_NODES: [Node] = [
         kind: .token(choices: [.keyword(text: "repeat")])
       ),
       Child(
-        name: "PatternType",
+        name: "Pack",
+        deprecatedName: "PatternType",
```

Comment:
@hborla I renamed the underlying type in `PackExpansionTypeSyntax` from `patternType` to `pack`. Does this make sense to you. `PatternType` didn’t make sense to me because the expression after `repeat` isn’t a pattern in the Swift pattern matching sense and it’s not really a pattern here.